app/amp_start.py

```diff
@@ -22,5 +22,7 @@ class Handler(start.Handler):
 
     def get(self):
         self.env.amp = True
+        """The reason for setting params.lang instead of env.lang is to prevent 
```

Comment:
This should be a normal comment, starting with "#", not a docstring """...""".

Docstring is only for a document of a class or a method.

Reply:
done.